router/MiddlewareLogger.go

```diff
@@ -0,0 +1,22 @@
+package router
+
+import (
+	"log"
+	"net/http"
+	"time"
+)
+
+func MiddlewareLogger(next http.Handler) http.Handler {
```

Comment:
Missing documentation.
Rename to `RequestTimeLoggerHandler`
Move into an handler folder